### ISSUE TITLE
Could not find com.android.tools.build:gradle:3.2.1 with plugin v.0.4.7

### ISSUE THREAD

**Issue by lgawin** [gladed/gradle-android-git-version#55]
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring project ':app'.
> Could not resolve all artifacts for configuration ':app:classpath'.
   > Could not find com.android.tools.build:gradle:3.2.1.
     Searched in the following locations:
       - https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.2.1/gradle-3.2.1.pom
       - https://plugins.gradle.org/m2/com/android/tools/build/gradle/3.2.1/gradle-3.2.1.jar
     Required by:
         project :app > com.gladed.androidgitversion:com.gladed.androidgitversion.gradle.plugin:0.4.7 > gradle.plugin.com.gladed.gradle.androidgitversion:gradle-android-git-version:0.4.7
```

It fails on command line (CI) but works with no problems when building in Android Studio

**Comment by lgawin** [comment]
So Android Studio seems to use one another (internal) repository when resolving dependencies for plugins
```
Attempting to resolve component for com.android.tools.build:gradle:3.2.1 
using repositories [maven, Gradle Central Plugin Repository]
```
where `maven` points to some local (file system) repository bundled with Android Studio (on Mac it is `/Applications/Android Studio X.Y.app/Contents/gradle/m2repository/` - where `X.Y` is the version of AS, there're some dependencies delivered and `gradle-3.2.1.jar` is there)

While when building from command line (`./gradlew ...`) it uses `Gradle Central Plugin Repository` only
```
Attempting to resolve component for com.android.tools.build:gradle:3.2.1 
using repositories [Gradle Central Plugin Repository]
```
and it cannot find the dependency there.

There're two solutions:
1. Do not use `plugins` DSL in `build.gradle`, so do it "old-school way"

In `app/build.gradle` use:
```
buildscript {
    repositories {
        gradlePluginPortal()
        google()
    }
    dependencies {
        classpath "gradle.plugin.com.gladed.gradle.androidgitversion:gradle-android-git-version:0.4.7"
    }
}
...
apply plugin: "com.gladed.androidgitversion"
...
```

2. Use `plugins` DSL and configure repositories

In `app/build.gradle` use:
```
plugins {
    id 'com.gladed.androidgitversion' version '0.4.7'
}
...
```
and in `settings.gradle` (in root) put at the top
```
pluginManagement {
    repositories {
        gradlePluginPortal()
        google()
    }
}
 ...
```

**Comment by gladed** [comment]
Thanks for the issue report. After review, I think it's better to just remove the requirement on `com.android.tools.build:gradle` so that the dependency isn't there in the first place.

**Comment by dbacinski** [comment]
Fixed in 0.4.9: https://plugins.gradle.org/plugin/com.gladed.androidgitversion
